builds/e2e/templates/e2e-setup.yaml

```diff
@@ -122,7 +122,7 @@ steps:
       $context['installerPath'] = Convert-Path '$(Build.SourcesDirectory)/scripts/windows/setup'
     }
 
-    if ('$(arch)' -eq 'arm32v7')
+    if ('$(arch)' -eq 'arm32v7' -Or '$(arch)' -eq 'arm64v8')
```

Comment:
Is the term `arm64v8` more appropriate/correct than `aarch64`?

Reply:
I believe the two terms are equivalent. Seems like they're used interchangeably.